dictdiffer/utils.py

```diff
@@ -270,6 +271,11 @@ def are_different(first, second, tolerance):
         return not (first_is_nan and second_is_nan)
     elif isinstance(first, num_types) and isinstance(second, num_types):
         # two numerical values are compared with tolerance
-        return abs(first-second) > tolerance * max(abs(first), abs(second))
+        return not math.isclose(
```

Comment:
Can you please remove PY 2.7 from `setup.py` and make sure we support only PY 3.5+?

Reply:
Ah yes, sure. Sorry I didn't see that `math.isclose` was only introduced in py35.
It should be good now. I had to set a constraint on the selenium version to make it work locally but I am not sure it is mandatory.

Reply:
Is the `_compat.py` then obsolete as well?

Reply:
I cleaned a few things in `_compat.py` and tested manually with py35. It needs a few tweaks to work with py35 so I did not include it in the CI (one must use `pytest-invenio<1.4.0` and change the `addopts` in `pytest.ini`).

Reply:
I would say we should remove the dependency in `pytest-invenio` and list the packages manually.

Reply:
If we don't use any feature of `pytest-invenio` is used it would be better indeed. I am going to check that.